### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by RomeoKs**
[DEMO LINK](https://RomeoKs.github.io/react_static-list-of-posts/)

### CODE REVIEW

**Review by Lun4trick — APPROVED** [review]
👍 GG

File: README.md [hunk 1]
```diff
@@ -17,4 +17,4 @@ This task is similar to [Static List of TODOs](https://github.com/mate-academy/r
 - Implement a solution following the [React task guideline](https://github.com/mate-academy/react_task-guideline#react-tasks-guideline).
 - Use the [React TypeScript cheat sheet](https://mate-academy.github.io/fe-program/js/extra/react-typescript).
 - Open one more terminal and run tests with `npm test` to ensure your solution is correct.
-- Replace `<your_account>` with your Github username in the [DEMO LINK](https://<your_account>.github.io/react_static-list-of-posts/) and add it to the PR description.
+- Replace `<your_account>` with your Github username in the [DEMO LINK](https://RomeoKs.github.io/react_static-list-of-posts/) and add it to the PR description.
```
Comment (meforoma) [inline README.md:20]: @RomeoKs i can't see the deployed page 
no gh pages deployments

**Review by meforoma — APPROVED** [review]
loved it 👍🏼 
just please make sure to deploy the page, kk?

**Comment by RomeoKs** [comment]
> loved it 👍🏼 just please make sure to deploy the page, kk?

It seems that I had an error while deploying.

image

Tried to  Enable Win32 long paths, but it's not working.
I'm still trying to find a solution.

**Comment by RomeoKs** [comment]
> > loved it 👍🏼 just please make sure to deploy the page, kk?
> 
> It seems that I had an error while deploying.
> 
> image
> 
> Tried to Enable Win32 long paths, but it's not working. I'm still trying to find a solution.

Solved it.
Thank you @meforoma

**Pull request closed without merging**